src/graphics/C4Shader.cpp

```diff
@@ -265,9 +265,9 @@ int C4Shader::ParsePosition(const char *szWhat, const char **ppPos)
 
 	// Lookup name
 	int iPosition = -1;
-	for (unsigned int i = 0; i < sizeof(C4SH_PosNames) / sizeof(*C4SH_PosNames); i++) {
-		if (SEqual(Name.getData(), C4SH_PosNames[i].Name)) {
-			iPosition = C4SH_PosNames[i].Position;
+	for (auto & C4SH_PosName : C4SH_PosNames) {
+		if (SEqual(Name.getData(), C4SH_PosName.Name)) {
+			iPosition = C4SH_PosName.Position;
```

Comment:
Will do.

I'm still not used to the naming convention - especially using first letter capital for variable names, it'll take some time for me to adjust.